app/templates/src/test/javascript/_protractor.conf.js

```diff
@@ -11,7 +11,7 @@ exports.config = {
     ],
 
     capabilities: {
-        'browserName': 'chrome',
```

Comment:
I mentioned why in https://github.com/mraible/generator-jhipster/commit/81022c740f2c8c4a8d2ff0395e31b3db15e2814c:

> Changed browser to Firefox by default since I'm guessing most developers are using Chrome. By testing in Firefox, you can 1) verify everything works in another browser and 2) continue using Chrome w/o Protractor grabbing your browser window.

> Also, Firefox seems to work better when running on a headless Linux box with Jenkins.